src/index.html

```diff
@@ -1,13 +1,57 @@
 <!doctype html>
-<html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <meta name="viewport"
-        content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
-  <title>Product cards</title>
-  <link rel="stylesheet" href="./style.css">
-</head>
-<body>
-  <h1>Product cards</h1>
-</body>
+<html lang="en" class="page">
+  <head>
+    <meta charset="UTF-8">
+    <meta name="viewport"
+          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <title>Product cards</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+      crossorigin="anonymous">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap"
+      rel="stylesheet">
+    <link rel="stylesheet" href="./styles/style.css">
+  </head>
+  <body>
+    <div class="product-card" data-qa="card">
+      <div class="product-card__container-img">
+        <img
+        src="/images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+        class="product-card__img"
+        >
+      </div>
+      <div class="product-card__info">
+        <h2 class="product-card__info-title">
+          APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+        </h2>
+        <p class="product-card__info-code">Product code: 195434</p>
+      </div>
+      <div class="product-card__rating-container">
+        <div class="stars">
+          <div class="stars__star"></div>
+          <div class="stars__star"></div>
```

Comment:
checklist point 3

Reply:
this one line, no multiline, if i correct understand
![image](https://user-images.githubusercontent.com/37651667/207063080-dab6ba77-1679-40b8-b73b-8bcf5efb9344.png)

Reply:
agree)